Assets/00_Content/Scripts/Rounds/RoundController.cs

```diff
@@ -10,33 +11,44 @@ public class RoundController : SingletonBehaviour<RoundController> {
 		[SerializeField] private ScalingData[] playerDifficulties;
 		[SerializeField] private Tavern tavern;
 		[SerializeField] private VikingController vikingController;
+		[SerializeField] private ScoreCard scoreCardPrefab;
 		[SerializeField, Tooltip("seconds/round")]
 		private int roundDuration;
 
+		private ScoreCard scoreCard;
 		private float roundTimer;
 		private int currentRound = 1;
+		private bool hasWon;
```

Comment:
Would roundIsActive or something similar be a more descriptive name? It feels a bit weird toggling if you have won during the same session.

Reply:
Mhm yeah, much better, I've renamed it to "isRoundActive"